task_8/sergey.bendas/src/test/java/com/hillel/TaskTwoTest.java

```diff
@@ -0,0 +1,103 @@
+package com.hillel;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class TaskTwoTest {
+
+  @Test
+  public void shouldFillSpiralArray() {
+    int[][] result = TaskTwo.fillSpiralArray(5, 5);
+    int[][] testArray = {
+        {1, 2, 3, 4, 5},
+        {16, 17, 18, 19, 6},
+        {15, 24, 25, 20, 7},
+        {14, 23, 22, 21, 8},
+        {13, 12, 11, 10, 9}
+    };
+    Assert.assertArrayEquals(testArray, result);
+  }
+
+  @Test
+  public void shouldFillSpiralArray2() {
+    int[][] result = TaskTwo.fillSpiralArray(1, 1);
+    int[][] testArray = {{1}};
+    Assert.assertArrayEquals(testArray, result);
+  }
+
+  @Test
+  public void shouldFillSpiralArray3() {
+    int[][] result = TaskTwo.fillSpiralArray(1, 2);
+    int[][] testArray = {{1, 2}};
+    Assert.assertArrayEquals(testArray, result);
+  }
+
+  @Test
+  public void shouldFillSpiralArray4() {
+    int[][] result = TaskTwo.fillSpiralArray(2, 2);
+    int[][] testArray = {
+        {1, 2},
+        {4, 3}
+    };
+    Assert.assertArrayEquals(testArray, result);
+  }
+
+  @Test
+  public void shouldFillSpiralArray5() {
+    long before = System.nanoTime();
+    int[][] result = TaskTwo.fillSpiralArray(6, 13);
+    long after = System.nanoTime();
+    System.out.println(after-before);
```

Comment:
You just need to place spaces before and after the **-** sign to pass code style check.